modules/organization/iam.tf

```diff
@@ -42,10 +42,11 @@ resource "google_organization_iam_custom_role" "roles" {
 }
 
 resource "google_organization_iam_binding" "authoritative" {
-  for_each = local.iam
-  org_id   = local.organization_id_numeric
-  role     = each.key
-  members  = each.value
+  for_each   = local.iam
+  org_id     = local.organization_id_numeric
+  role       = each.key
+  members    = each.value
+  depends_on = [google_organization_iam_custom_role.roles]
```

Comment:
I'm not entirely sure about this. You need the role admin role to be able to create a custom role, so this dependency might raise issues in specific cases. I remember looking at this for FAST, and deciding the custom roles should depend in bindings, but not viceversa. Remember we're at org level, so unless you have role admin role, you cannot create custom roles.

Reply:
This dependency is "not null", only if you create custom roles. Custom role doesn't have any explicit dependencies, so it won't loop back here. In usual case, this will be empty, so I assume this should not raise any issues.

Otherwise this example:
```hcl
module "org" {
  source          = "./fabric/modules/organization"
  organization_id = var.organization_id
  custom_roles = {
    "myRole" = [
      "compute.instances.list",
    ]
  }
  iam = {
    (module.org.custom_role_id.myRole) = ["group:${var.group_email}"]
  }
}
```
Should be considered as "not supported use", because this may fail.

I can give a FAST end-to-end test on this branch to check if it breaks anything, WDYT?

Reply:
yep, the part I'm worried about is not assigning a custom role, but creating one at org level which requires having the role admin role assigned

Reply:
if both fail with/without, priority goes to the FAST use case ofc (so no dependency)

Reply:
rolled back this change. I realized that the example provides explicit dependency on the role so it should work (and it is, as far as I tested).

For FAST - we have prerequisite that when running `0-bootstrap` the identity running bootstrap already has organization role admin.